lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -1,4 +1,6 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
+    expose :validate_import_vm
+    expose :import_vm
```

Comment:
@matobet 
What class would the import_vm return, just want to make sure that its not another vmdb object that needs to be from the service model.